modules/3d/src/octree.cpp

```diff
@@ -348,45 +376,78 @@ bool deletePointRecurse(Ptr<OctreeNode>& _node)
     }
 }
 
-bool insertPointRecurse( Ptr<OctreeNode>& _node,  const Point3f& point, int maxDepth)
+bool insertPointRecurse( Ptr<OctreeNode>& _node,  const Point3f& point,const Point3f &color, size_t maxDepth,const OctreeKey &key,
```

Comment:
Looks like this function always returns true, don't need to return anything at all